sdcm/ndbench_thread.py

```diff
@@ -91,10 +91,9 @@ def __init__(self, *args, **kwargs):
                           f' -Dndbench.config.cass.host={self.node_list[0].external_address} {self.stress_cmd} run'
 
     def _run_stress(self, loader, loader_idx, cpu_idx):
-        log_dir = os.path.join(loader.logdir, self.loader_set.name)
-        if not os.path.exists(log_dir):
-            os.makedirs(log_dir)
-        log_file_name = os.path.join(log_dir, f'ndbench-l{loader_idx}-c{cpu_idx}-{uuid.uuid4()}.log')
+        if not os.path.exists(loader.logdir):
```

Comment:
create a task that generalizes this logic

Reply:
fixed